routes/confirmation/xmlFormat.js

```diff
@@ -0,0 +1,450 @@
+  },
+]
+
+const lineShouldBeAdded = (line, session) => {
+  return !Object.prototype.hasOwnProperty.call(line, 'addIf') || (Object.prototype.hasOwnProperty.call(line, 'addIf') && hasData(session, line.addIf))
+}
+
+const outputXML = (req, createFile = false, linesToAdd = dataToLine) => {
+  // force a deep copy to keep it clean between runs
+  const newXml = JSON.parse(JSON.stringify(templateXml))
+
+  const linesForEdit = newXml['elements'][0]['elements'][0]['elements'][0]['elements']
+  
+  linesForEdit.push(addTaxPayerInfo(req.session.personal, req.locale))
+
+  linesToAdd.map((originalLine) => {
+    let line = Object.assign({}, originalLine)
+    let newLine = {}
+
+    /**
+     * some initial thoughts: right now i have a case by line
+     * but this can be unruly. I think in the future, we'll probably want in the dataToLine some sort of options, that will specify this value is meant to be multiplied, or check these values for a single true, etc
+     * but without an exact idea of how we should format this, i'll leave it as is for now and we'll revisit
+     */
+    switch(true) {
+      case line.line === 5884: {
+        const initialValue = hasData(req.session, line.value, true)
+
+        const multipliedValue =  Math.round(initialValue * 0.0505)
+
+        newLine = createReturnLine(line.line, multipliedValue)
+
+        return linesForEdit.push(newLine)
+      }
+      case line.line === 6118: {
+
+        const fieldValue = ifAnyTrue(line, req.session)
+
+        newLine = createReturnLine(line.line, fieldValue)
+
+        return linesForEdit.push(newLine) 
+      }
+      case line.value.toLowerCase().includes('amount'): {
+        /**
+         * another note: we may not always want to round
+         * it seems that sometimes they just round, and other times they just remove a decimal
+         * so $20.48 will be passed as 2048
+         * and then probably somewhere on there system they just know that the last 2 digits fall after a decimal
+         * but again, without exact knowledge of the formatting we need, and since it's MOSTLY rounded, we round for our simple base purposes
+         */
+
+        if (lineShouldBeAdded(line, req.session)) {
+
+          const initialValue = hasData(req.session, line.value, true)
+
+          const roundedValue = Math.round(initialValue)
+
+          newLine = createReturnLine(line.line, roundedValue)
+          
+          return linesForEdit.push(newLine) 
+        }
+
+        break
+
+      }
+      default:
+        if (lineShouldBeAdded(line, req.session)) {
+          const returnValue = hasData(req.session, line.value, true) === true ? 1 :0
+
+          newLine = createReturnLine(line.line, returnValue)
+
+          return linesForEdit.push(newLine)
+        }
+
+        break
+    }
+  })
+  
+
+  const data = convert.json2xml(newXml , {spaces: 2})
+
+  // this because I don't want an output everytime we run the spec files
+  if(createFile) {
```

Comment:
We can look for `process.env.NODE_ENV !== 'test'` or something like that. 

`npm test` and the `npm run cypress*` commands both run in "test" mode.